### PR TITLE
Make more files printable

### PR DESCRIPTION
**Pull request by thisIsTheFoxe**
- #1865 

Printable files:
- PDFs
- Images (jpg, png, etc.)
- `svg` images
- any _text_ file (txt, md, html)

### CODE REVIEW

**Comment by marinofaggiana** [comment]
No time for test, works ?

File: iOSClient/Data/NCDatabase.swift [hunk 1]
```diff
@@ -415,6 +416,10 @@ class tableMetadata: Object, NCUserBaseUrl {
 
 extension tableMetadata {
     var fileExtension: String { (fileNameView as NSString).pathExtension }
+
+    var isPrintable: Bool {
+        classFile == NCCommunicationCommon.typeClassFile.image.rawValue || ["application/pdf", "com.adobe.pdf"].contains(contentType) || contentType.hasPrefix("text/")
+    }
```
Comment (marinofaggiana) [inline iOSClient/Data/NCDatabase.swift:422]: var fileExtension: String { (fileNameView as NSString).pathExtension } ?
Already exists the ext in  tableMetadata

**Review by marinofaggiana — CHANGES_REQUESTED** [review]
Read comment.

File: iOSClient/Data/NCDatabase.swift (hunk 1 above)
Comment (thisIsTheFoxe) [reply iOSClient/Data/NCDatabase.swift:422]: Yes, but that can be used if `ext` is not set or empty e.g. in the RenameFile. See also #1885

**Comment by thisIsTheFoxe** [comment]
I wanted to do some more tests, but if you think it works feel free to merge..

**Review by marinofaggiana — APPROVED** [review]

**Pull request merged**